pkg/prove/proof_test.go

```diff
@@ -170,61 +170,6 @@ func TestTxShareIndex(t *testing.T) {
 	}
 }
 
-// TODO: Uncomment/fix this test after we've adjusted tx inclusion proofs to
```

Comment:
I think we should create an issue to fix this one as it seems important

Reply:
an alternative to just deleting would be to just replace the test code with `t.Skip("this test needs to be re-implemented")` 

but if the goal is to eventually redo these tests, we should definitely have issues for them

Reply:
[blocking] 
if we don't have an issue for this, lets create one and merge

Reply:
Created https://github.com/celestiaorg/celestia-app/issues/1049